packages/modules/manage-versions/src/components/ManageVersions/VersionsTab/VersionsTab.tsx

```diff
@@ -66,6 +67,36 @@ const VersionsTab = (props: VersionsTabProps) => {
     }
   }
 
+  const onDownloadClick = useCallback(
+    async (changesetIndex: number) => {
+      toaster.closeAll();
+      try {
+        const checkpointInfo = await changesetClient.getChangesetCheckpoint(
+          imodelId,
+          changesetIndex
+        );
+        const downloadUrl = checkpointInfo._links.download.href;
+        window.open(downloadUrl, "_blank", "noopener,noreferrer");
+        toaster.positive(stringsOverrides.messageDownloadedFileSuccessfully, {
```

Comment:
ok, last thing.. Do you think it makes sense to show positive toast on download click?
It pops up browser window with download... and you dont really check the progress (what if it takes longer?? your message then would be even invalid..)